sdk/web/armweb/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Release History
+
+## v0.1.0 (released)
```

Comment:
This should be a date of release i.e (2021-08-09)

Reply:
We can have more discussion on how to write the changelog and compose a document accordingly maybe?
But here we are going to release this now...